### PR TITLE
Apiv2 upgrade

### PR DESCRIPTION
**Pull request by wkdewey**

### CODE REVIEW

**Review by techgique — CHANGES_REQUESTED** [review]

File: app/views/layouts/head/_stylesheets.html.erb [hunk 1]
```diff
@@ -1,4 +1,5 @@
   <!-- Stylesheets -->
+  <%# include Orchid::ApplicationHelper %>
```
[the remark this answers is not in the document]
Comment (wkdewey) [reply app/views/layouts/head/_stylesheets.html.erb:2]: I think it may have been a mistake

[the remark this answers is not in the document]
Comment (wkdewey) [reply app/views/layouts/head/_stylesheets.html.erb:2]: I was probably trying to debug something and though I needed that include line. I'll take it out.

[the remark this answers is not in the document]
Comment (wkdewey) [reply app/views/layouts/head/_stylesheets.html.erb]: removed (looks like the `include` needed to be handled in config instead)

File: config/environments/production.rb [hunk 2]
```diff
@@ -46,11 +46,17 @@
...
+  config.force_ssl = ENV['RAILS_PROD_NOSSL'].blank?
+  #   Handle STS here instead of Apache, or Rails duplicates header contents
+  #   Also unset cache-control header in HTTPS vhost for same reason
+  config.ssl_options = { hsts: { preload: true } }
 
   # Include generic and useful information about system operation, but avoid logging too much
   # information to avoid inadvertent exposure of personally identifiable information (PII).
-  config.log_level = :info
+  # Local change to reduce log bloat
+  config.log_level = :warn
```
[the remark this answers is not in the document]
Comment (wkdewey) [reply config/environments/production.rb]: done

**Review by techgique — APPROVED** [review]

**Pull request merged**